blaze-client/src/test/scala/org/http4s/client/blaze/Http1ClientStageSpec.scala

```diff
@@ -13,14 +13,13 @@ import scodec.bits.ByteVector
 
 import scala.concurrent.Await
 import scala.concurrent.duration._
-import scalaz.\/-
-import scalaz.concurrent.{Strategy, Task}
+import fs2._
 
 // TODO: this needs more tests
 class Http1ClientStageSpec extends Http4sSpec {
 
   val ec = org.http4s.blaze.util.Execution.trampoline
-  val es = Strategy.DefaultExecutorService
+  val es = impl.DefaultExecutor.newClientDefaultExecutorService("Http1ClientStageSpec")
```

Comment:
Same problem as above.  We need to shut down the non-defaults or reuse the `Http4sSpec` one.